README.md

```diff
@@ -1 +1,185 @@
+  <img src="src/images/logo.png" alt="logo" width="140"  height="auto"  />
+  <br/>
+
+  <h2><b>Restaurant</b></h2>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Restaurant] <a name="about-project"></a>
+
+**[Our Restaurant]** is a website for the restaurant. With this app , customers can see the meals , they can like and comment on meals. All data is preserved thanks to [Involvement API](https://microverse.notion.site/Involvement-API-869e60b5ad104603aa6db59e08150270). This website is build by using JavaScript, Webpack and ES6.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Javascript
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[display the meals]**
+- **[add like]**
+- **[add comment]**
+- **[display the likes and comments]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+>
+- [Live Demo](https://abutalha3.github.io/csp-js-module2/)
+
+<!-- GETTING STARTED -->
+
+>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ npm install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/AbuTalha3/csp-js-module2.git
+
+  cd javascript-capstone
+```
+
+### Install
+
+Install this project with:
+
+```sh
+ npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run build
+```
+
+### Run tests
+
+To run tests on linters, run the following command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx hint .
+  npx eslint .
+```
+
+To run test coverages, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+   npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Siddhartha Ghosh**
+
+- GitHub: [@siddghosh108](https://github.com/siddghosh108)
+- Twitter: [@siddharthaghos9](https://twitter.com/siddharthaghos9)
+- LinkedIn: [@siddhartha-ghosh-65902718](https://www.linkedin.com/in/siddhartha-ghosh-65902718)
```

Comment:
- [ ] Seeing that it is a group project, I kindly advice that you add your partners profile as one of the authors of this project, that way he equally gets recognition for his contributions 👍

Reply:
Includer my partner Abu Talha as an Author in README.md

👤 **Abu Talha**

- GitHub: [@githubhandle](https://github.com/AbuTalha3)
- Twitter: [@twitterhandle](https://twitter.com/AbuTalha8T)
- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abu-talha-8203b252/)